src/actions/index.js

```diff
@@ -0,0 +1,41 @@
+var options = {
+    headers: {
+                "Authorization": "token // paste your token here // "
+             }
```

Comment:
Ah, I think I get it now ... does this authorize people to change the update the redux stores?